src/app-simplefin/app-simplefin.js

```diff
@@ -210,12 +210,13 @@ function getAccountResponse(results, accountId, startDate) {
       dateToUse = trans.posted;
     }
 
-    newTrans.bookingDate = getDate(new Date(dateToUse * 1000));
```

Comment:
Does SF not use `bookingDate`?

Reply:
No, all bank sync looks to use `date` https://github.com/actualbudget/actual/blob/c626fc2f17199a327c8dcd80e93c3549e8926240/packages/loot-core/src/server/accounts/sync.ts#L326